packages/react-ui-core/src/Carousel/PhotoCarousel.js

```diff
@@ -12,13 +12,24 @@ export default class PhotoCarousel extends PureComponent {
   static propTypes = {
     className: PropTypes.string,
     theme: PropTypes.object,
-    items: PropTypes.arrayOf(
-      PropTypes.shape({
-        caption: PropTypes.string,
-        path: PropTypes.string,
-        url: PropTypes.string,
-      }),
-    ),
+    items: PropTypes.oneOfType([
+      PropTypes.arrayOf(
+        PropTypes.arrayOf(
+          PropTypes.shape({
+            caption: PropTypes.string,
+            path: PropTypes.string,
+            url: PropTypes.string,
+          }),
```

Comment:
Can we make this into a type (e.g. `CarouselPhoto`)  instead of copying it?

```js
          PropTypes.shape({
            caption: PropTypes.string,
            path: PropTypes.string,
            url: PropTypes.string,
          }),
```

Reply:
👍